servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -1883,6 +1891,7 @@ private void verifyMetadataProperties(
     assertThat(referenceMetadata.numCommitsBehind()).isEqualTo(expectedCommitsBehind);
     assertThat(referenceMetadata.commitMetaOfHEAD()).isEqualTo(commitMeta);
     assertThat(referenceMetadata.commonAncestorHash()).isEqualTo(reference.getHash());
+    assertThat(referenceMetadata.numTotalCommits()).isEqualTo(expectedCommits);
```

Comment:
needs to have a `isNull` assertion in `verifyMetadataProperties(Tag tag)`

Reply:
fixed